### ISSUE TITLE
TypeError: unhashable type: 'dict'

### ISSUE THREAD

**Issue by porowns** [Kyria/EsiPy#11]
Whenever querying mails, it seems to break.

```
esiclient = EsiClient(
    security=esi_security,
    cache=None,
    headers={'User-Agent': 'example'}
)

op = esi_app.op['get_characters_character_id_mail'](character_id=c_id)

mails = esiclient.request(op)
```

**Comment by Kyria** [comment]
regarding to the error stack, it looks like it's a pyswagger issue :/ 
i'll open a issue there

**Comment by Kyria** [comment]
Also, as an alternative that works, until this is fixed, you can do the following:

```python
mail_request = esiclient.request(op, raw_body_only=True)
import json
mails = json.loads(mails.raw)
```

So you'll have at least your data.

**Comment by Kyria** [comment]
@porowns This should be fixed with the latest pyswagger version v0.8.30.
